src/redux/reducers/ConceptReducers.js

```diff
@@ -1,28 +1,49 @@
-import { IS_FETCHING, FETCH_CONCEPTS, FETCH_DICTIONARY_CONCEPT } from '../actions/types';
-import { filterSources, filterClass } from './util';
+import * as types from '../actions/types';
```

Comment:
Did you take a look at this? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports

Reply:
Fixed. @dkayiwa